muse_payment_adaptor/api_interface/muse_http_client.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from typing import Any
+from urllib.parse import urljoin, urlencode
+
+import requests
+from requests import Response
+
+from muse_payment_adaptor.apps import MusePaymentAdaptorConfig
+
+logger = logging.getLogger(__name__)
+
+
+class MuseAdaptorHttpClient:
+    """
+    Http client to be used for muse http requests
+    """
+
+    def __init__(self, url: str):
+        """
+        :param url: Root url for all request from this client
+        """
+        self.url = url
+
+    def send_post_request(self, path: str, body: Any) -> Response:
```

Comment:
Please change `path` to more descriptive name. Will `endpoint_uri` work?

Reply:
changed to endpoint_url and root_url for init arg